mcp/pacemaker.sysconfig

```diff
@@ -1,95 +1,102 @@
 
-# Log all messages from a comma-separated list of files (no path)
-# Supports wildcards eg. PCMK_trace_files=prefix*.c
+# Log all messages from a comma-separated list of files (no path).
+# Supports wildcards eg. PCMK_trace_files=prefix*.c.
 # PCMK_trace_files=file.c,other.h
 
-# Log all messages matching comma-separated list of formats
+# Log all messages matching comma-separated list of formats.
 # PCMK_trace_formats="Sent delete %d"
 
-# Log all messages from a comma-separated list of tags
+# Log all messages from a comma-separated list of tags.
 # PCMK_trace_tags=tag1,tag2
 
 # Dump the blackbox whenever the message at function and line is printed
-# eg. PCMK_trace_blackbox=te_graph_trigger:223,unpack_clone:81
+# eg. PCMK_trace_blackbox=te_graph_trigger:223,unpack_clone:81.
 # PCMK_trace_blackbox=fn:line,fn2:line2,...
 
-# Enable blackbox logging globally or per-subsystem
-# The blackbox contains a rolling buffer of all logs (including info+debug+trace)
-# and is written after a crash, assertion failure and/or when SIGTRAP is received
+# Enable blackbox logging globally or per-subsystem.
+# The blackbox contains a rolling buffer of all logs (incl. info+debug+trace)
+# and is written after a crash, assertion failure and/or when SIGTRAP is
+# received.
 #
-# The blackbox recorder can also be enabled for Pacemaker daemons at runtime by 
-# sending SIGUSR1 (or SIGTRAP), and disabled by sending SIGUSR2
+# The blackbox recorder can also be enabled for Pacemaker daemons at runtime
+# by sending SIGUSR1 (or SIGTRAP), and disabled by sending SIGUSR2.
 #
 # Multiple subsystems may me listed separated by commas
-# eg. PCMK_blackbox=crmd,pengine
+# eg. PCMK_blackbox=crmd,pengine.
 # PCMK_blackbox=yes|no|crmd|pengine|cib|stonith-ng|attrd|pacemakerd
 
 #==#==# Advanced use only
 
 # Enable this for compatibility with older corosync (prior to 2.0)
-# based clusters which used the nodes uname as its uuid also
+# based clusters which used the nodes uname as its uuid also.
 # PCMK_uname_is_uuid=no
 
-# Specify an alternate location for RNG schemas and XSL transforms
-# Mostly only useful for developer testing
+# Specify an alternate location for RNG schemas and XSL transforms.
+# Mostly only useful for developer testing.
 # PCMK_schema_directory=/some/path
 
-# Enable this for rebooting this machine at the time of process (subsystem) failure
+# Enable this for rebooting this machine at the time of process (subsystem)
+# failure.
 # PCMK_fail_fast=no
 
-# Set action at the time of the system reboot by fail_fast
-# When set 'crash' , the kernel does panic
-# When you want to get kdump , please set 'crash'
-# The default action is 'reboot'
+# Set action at the time of the system reboot by fail_fast.
+# When set 'crash' , the kernel does panic.
+# When you want to get kdump , please set 'crash'.
+# The default action is 'reboot'.
 # PCMK_panic_action=crash
 
 #==#==# Pacemaker Remote
 # Use a custom directory for finding the authkey.
```

Comment:
Maybe: 'Use the contents of this file as the authorization key to use with Pacemaker Remote connections. This file must be readable by Pacemaker daemons (that is, it must allow read permissions to either the hacluster user or the haclient group), and its contents must be identical on all nodes. The default is "/etc/pacemaker/authkey".' Of course, we should substitute the configured user/group.